pkg/local_object_storage/internal/log/log.go

```diff
@@ -24,3 +24,13 @@ func AddressField(addr interface{}) zap.Field {
 func OpField(op string) zap.Field {
 	return zap.String("op", op)
 }
+
+// StorageTypeField returns logger's field for storage type.
+func StorageTypeField(typ string) zap.Field {
+	return zap.String("type", typ)
+}
+
+// StorageIDField returns logger's field for storage ID.
+func StorageIDField(id []byte) zap.Field {
+	return zap.String("storageID", string(id))
```

Comment:
Let's stay on `storage_id` in logs? Like we already do for `shard_id`

Reply:
agree, changed